lib/enrich.ts

```diff
@@ -0,0 +1,100 @@
+import { EMBED_JSONLD_CONTEXT } from '../config';
+import { OPARL_JSON_LD_CONTEXT } from '../constants';
+import { removeVersionFromOparlSchemaUri, rewriteLinkWithProxy } from './utils';
+
+export function enrichOparlDataToJsonLd(oparlData: any, proxyUrl: string) {
+  let dataBlock = [];
+  if (oparlData['data']) {
+    dataBlock = oparlData['data'];
+  } else {
+    dataBlock = [oparlData];
+  }
+  let linksBlock = {
+    id: proxyUrl,
+    type: 'Node',
+  };
+  if (oparlData['links']) {
+    linksBlock = { ...linksBlock, ...oparlData['links'] };
+  }
+  // Add linkToPublications based on content
```

Comment:
for instance, if we make these comments functions, the code starts documenting itself